src/i18n/en.json

```diff
@@ -2137,6 +2137,8 @@
     "select_template": "Select a Email Template to perform the action.",
     "select_items": "Select at least one invitation to perform the action.",
     "tags": "Tags",
+    "items_qty": "Selected {qty} Invitations",
+    "send_email_warning": "You're about to send the invite email template to {qty} people. Please confirm. ",
```

Comment:
please correct his to be 
"You're about to send the {template} email template to {qty} people. Please confirm. ",
and set the proper value to the template variable using value selected on flow_event DDL